### PR TITLE
[Hotfix][Connector-V2][Jdbc] Fix convert null value to long type exception

### PR DESCRIPTION
**Pull request by harveyyue**
Fix parsing null value exception.

## Check list

* [ ] Code changed are covered with tests, or it does not need tests for reason:
* [ ] If any new Jar binary package adding in your PR, please add License Notice according
  [New License Guide](https://github.com/apache/incubator-seatunnel/blob/dev/docs/en/contribution/new-license.md)
* [ ] If necessary, please update the documentation to describe the new feature. https://github.com/apache/incubator-seatunnel/tree/dev/docs
* [ ] If you are contributing the connector code, please check that the following files are updated:
  1. Update change log that in connector document. For more details you can refer to [connector-v2](https://github.com/apache/incubator-seatunnel/tree/dev/docs/en/connector-v2)
  2. Update [plugin-mapping.properties](https://github.com/apache/incubator-seatunnel/blob/dev/plugin-mapping.properties) and add new connector information in it
  3. Update the pom file of [seatunnel-dist](https://github.com/apache/incubator-seatunnel/blob/dev/seatunnel-dist/pom.xml)
* [ ] Update the [`release-note`](https://github.com/apache/incubator-seatunnel/blob/dev/release-note.md).

### CODE REVIEW

**Comment by liugddx** [comment]
To zero may not be good choice. Or direct exception?

**Comment by harveyyue** [comment]
> To zero may not be good choice. Or direct exception?

Actually, we expect the job running successfully and print some warn message, not throw exception

File: seatunnel-connectors-v2/connector-jdbc/src/main/java/org/apache/seatunnel/connectors/seatunnel/jdbc/source/JdbcSource.java [hunk 1]
```diff
@@ -163,10 +163,10 @@ private PartitionParameter initPartitionParameter(String columnName, Connection
             if (rs.next()) {
                 max = jdbcSourceOptions.getPartitionUpperBound().isPresent() ?
                     jdbcSourceOptions.getPartitionUpperBound().get() :
-                    Long.parseLong(rs.getString(1));
+                    rs.getString(1) == null ? 0L : Long.parseLong(rs.getString(1));
```
Comment (EricJoy2048) [inline seatunnel-connectors-v2/connector-jdbc/src/main/java/org/apache/seatunnel/connectors/seatunnel/jdbc/source/JdbcSource.java]: I suggest use `Long.MAX_VALUE` and `Long.MIN_VALUE` when `rs.getString(1) == null` or `rs.getString(2) == null `. And then we should add warning log too.

Comment (harveyyue) [reply seatunnel-connectors-v2/connector-jdbc/src/main/java/org/apache/seatunnel/connectors/seatunnel/jdbc/source/JdbcSource.java]: done.

**Review by hailin0 — DISMISSED** [review]

**Comment by hailin0** [comment]
update title: [Hotfix][Connector-V2][Jdbc] Fix ...

**Comment by harveyyue** [comment]
> update title: [Hotfix][Connector-V2][Jdbc] Fix ...

done.

**Comment by hailin0** [comment]
Sorry, because the code structure is outdated

**Comment by hailin0** [comment]
This pr has been fixed
https://github.com/apache/incubator-seatunnel/pull/4570

**Pull request closed without merging**